src/Frontend/src/components/configuration/MassTransitConnector.vue

```diff
@@ -3,9 +3,9 @@ import { useConfiguration } from "@/composables/configuration";
 import moment from "moment";
 
 const configuration = useConfiguration();
-// "Wed, Jan 15th 2025 10:56:21 +10:00",
+
 function formatDate(date: string) {
-  return moment(date).local().format("LLLL"); //.format("ddd, MMM Do YYYY HH:mm:ss Z");
+  return moment(date).local().format("ddd, MMM Do YYYY HH:mm:ss Z");
```

Comment:
```suggestion
  return moment(date).local().format("LLLL");
```
this is the format used elsewhere